cli/core/pkg/artifact/interface_test.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2021 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package artifact
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = FDescribe("Unit tests for local artifact", func() {
```

Comment:
we could remove focused  `FDescribe` here.

Reply:
thanks for comment's, PR was in-progress, its been fixex.